.github/workflows/build-mudlet.yml

```diff
@@ -39,12 +39,19 @@ jobs:
             gcc_compiler_version: 9
             qt: '5.14.2'
           - os: macos-12
-            buildname: 'macos / c++, lua tests'
+            buildname: 'macos (x86_64) / c++, lua tests'
             triplet: x64-osx
             compiler: clang_64
             qt: '5.14.2'
             deploy: 'deploy'
             run_tests: 'true'
+          - os: macos-14
+            buildname: 'macos (arm64) / c++, lua tests'
+            triplet: arm64-osx
+            compiler: clang_64
+            qt: '5.15.13'
+            deploy: 'deploy'
+            run_tests: 'true'
```

Comment:
Do we want/need this bit?

Reply:
I would say yes

Reply:
not sure which bit you're referring to here? These are matrix variables that are depended on by downstream steps.